### PR TITLE
(fix) [SD-3510] Improve formatting of related items in the editor

### PR DESCRIPTION
**Pull request by fritzSF**

### CODE REVIEW

File: client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html [hunk 1]
```diff
@@ -31,12 +31,12 @@
 
     </div>
 
-    
+
    <!--  <div class="filter-box" ng-show="options.mode === 'detailed'">
         <span class="text" translate>View</span>
-        <button class="btn btn-light pull-left" 
+        <button class="btn btn-light pull-left"
```
Comment (pavlovicnemanja) [inline client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html]: Hey, why we left the commented part inside file? Do we need it for something?

Comment (fritzSF) [inline client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html]: i don't think so, the Ingest and Content widgets will be deprecated anyway

Comment (pavlovicnemanja) [inline client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html]: Maybe we can remove it, to keep code clean. This part of code was commented on April 7, 2015.

**Comment by petrjasek** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

File: client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html (hunk 1 above)
Comment (fritzSF) [inline client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html]: Sure, remove it.

**Pull request merged**